### PR TITLE
feature/vuex-setup

### PR DESCRIPTION
**Pull request by sletap**
You can login without entering credentials if you've logged in before on your device

### CODE REVIEW

**Comment by sletap** [comment]
This is super annoying right now it sorta works but it's very hard coded. Right now if you go to the landing page only Maya's name will work no matter who you're signed in as. I tried to abstract it out but I'm running into asynchronous related issues which are super super annoying. Will have to do more research

**Comment by sletap** [comment]
This is a VERY bad implementation because of asynchronous problems I'll have to go in to look at it again soon. Might have to go a different route cause this one is trash

File: src/App.vue [hunk 1]
```diff
@@ -35,6 +39,14 @@ export default {
       const fromDepth = from.path.split('/').length
       this.transitionName = toDepth > fromDepth ? 'slide-right' : 'slide-left'
     }
+  },
+  created(){
+    auth.onAuthStateChanged(function(user) {
+      if (user) {
+        // Add user to vuex store
+        store.commit('addUser', user)
```
Comment (sletap) [inline src/App.vue]: Maybe I can dispatch once here instead and do both in one function call to keep things together

Comment (Edward-Knighton) [reply src/App.vue]: Why cant we just use a ```.then``` and change the login button to a css loader while it calls (https://loading.io/css/) feel like  that is  pretty common app  behavior as long  as it doesn't take more than like 5 seconds

Comment (sletap) [reply src/App.vue]: Ooo yea that's a solid idea I'll try to do a little thing there

**Comment by sletap** [comment]
https://deploy-preview-12--ktpoints.netlify.com/landing

**Pull request merged**